spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -61,7 +70,7 @@ deployment to fail
 ```
 cf push dataflow-server --no-start -p spring-cloud-dataflow-server-cloudfoundry-{project-version}.jar
 cf bind-service dataflow-server redis
-cf bind-service dataflow-server rabbit
```

Comment:
redis is used by analytics query commands.
I don't understand why rabbit was here. does `task launch` talk to rabbit directly?